tofu/geom/utils.py

```diff
@@ -693,7 +693,7 @@ def create_config(case=None, Exp='Dummy', Type='Tor',
                   Lim=None, Bump_posextent=[np.pi/4., np.pi/4],
                   R=2.4, r=1., elong=0., Dshape=0.,
                   divlow=True, divup=True, nP=200,
-                  out='object', SavePath='./'):
+                  out='object', SavePath='./', path=_path_testcases):
     """ Create easily a tofu.geom.Config object
```

Comment:
sometimes I run tests from other paths, so I added to possibility to determine the path for testing